tests/functional/adapter/concurrent_transaction/test_concurrent_transaction.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import threading
+
+from dbt.tests.util import run_dbt
+from dbt.tests.adapter.simple_seed.test_seed import SeedConfigBase
+from tests.functional.adapter.concurrent_transaction.fixtures import *
+
+
+class BaseConcurrentTransaction(SeedConfigBase):
+    def __init__(self):
+        self.query_state = None
+
+    def reset(self):
+        self.query_state = {
+            'view_model': 'wait',
+            'model_1': 'wait',
+        }
+
+    @pytest.fixture(scope="class")
+    def setUp(self, project):
+        self.reset()
+
+    @pytest.fixture(scope="class")
+    def schema(self):
```

Comment:
Do we need to specify the schema? Or can we just use the random unique one that the framework creates for each run?

Reply:
Let me test if this isn't needed.